kvm/v12.0.0/README.md

```diff
@@ -0,0 +1,52 @@
+## :zap: Giant Swarm Release 12.0.0 for KVM :zap:
+
+This is the first Giant Swarm release which includes Kubernetes v1.17. Components have been updated for improved reliability and observability. Further details about changes to individual components can be found below.
+
+### Kubernetes v1.17.8
+
+- Updated from v1.16.9 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.17.md#changelog-since-v1177).
+- **Cloud Provider Labels reach General Availability**: Added as a beta feature way back in v1.2, v1.17 sees the general availability of cloud provider labels.
+- **Volume Snapshot Moves to Beta**: The Kubernetes Volume Snapshot feature is now beta in Kubernetes v1.17. It was introduced as alpha in Kubernetes v1.12, with a second alpha with breaking changes in Kubernetes v1.13.
+- **CSI Migration Beta**: The Kubernetes in-tree storage plugin to Container Storage Interface (CSI) migration infrastructure is now beta in Kubernetes v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
+
```

Comment:
Flatcar update might be worth mentioning somewhere as well

Reply:
For sure. It's there now.